src/js/actions/signalActions.ts

```diff
@@ -1,12 +1,5 @@
 import {createStandardAction} from 'typesafe-actions';
-import {OnEvent} from 'vega-typings/types';
-
-const ns = require('../util/ns');
-
-    INIT_SIGNAL = 'INIT_SIGNAL',
-    SET_SIGNAL  = 'SET_SIGNAL',
-    SET_SIGNAL_STREAMS = 'SET_SIGNAL_STREAMS',
-    UNSET_SIGNAL = 'UNSET_SIGNAL';
+import {OnEvent, SignalValue} from 'vega-typings/types';
```

Comment:
I'm curious about why we need to import from `vega-typings/types` rather than just `vega-typings` (or even `vega`)?

Reply:
all of the above will work, i think i just let vscode autocomplete pick every time. i will make it consistent